Sources/HipMobileUI.Droid/Contracts/AndroidFabSizeCalculator.cs

```diff
@@ -30,7 +32,23 @@ public int CalculateFabSize()
                 return 56;
             }
             // compat libraries use pixel, therefore convert dp to pixel
-            return (int) (56 * scale + 0.5);
+            return (int) (56 * density);
+        }
+
+        /// <summary>
+        /// Returns the os version number of the device. E.g. 4 for KitKat.
+        /// </summary>
```

Comment:
The documentation should be on the interface method definition, not the implementation

Reply:
Done